project/ISSUE-TRIAGE.md

```diff
@@ -86,19 +67,26 @@ labels.
 | Level            | Experience level guideline                                                                                               |
 |------------------|--------------------------------------------------------------------------------------------------------------------------|
 | exp/beginner     | You have made less than 10 contributions in your life time to any open source project.                                   |
```

Comment:
I'm not sure the experience level has anything to do with how many contributions you've made.  Rather its more about your level of understanding of docker and golang.  Perhaps something more like:
```
beginner: new to Docker, and possibly golang, and is looking to help while learning the basics.
intermediate: comfortable with golang and understands the core concepts of Docker and
looking to dive deeper into the project.
expert: proficient with Docker and golang and has been following, and active in, the community
to understand the rational behind many design decisions and where the project is headed.
```